subgraph/src/public-lock.ts

```diff
@@ -269,7 +272,11 @@ export function handleRenewKeyPurchase(event: RenewKeyPurchaseEvent): void {
   }
 
   // create receipt
-  createReceipt(event)
+  if (key && key.owner) {
+    createReceipt(event, key.owner, tokenId.value)
+  } else {
+    createReceipt(event, Bytes.empty(), tokenId.value)
```

Comment:
Same as above, when does a key without an owner happen in renewal? We should not pass empty Bytes but at least a null address